tools/pipelines/dependency-injection/.npmrc

```diff
@@ -4,5 +4,5 @@ always-auth=false
 @fluidframework:registry=https://pkgs.dev.azure.com/fluidframework/internal/_packaging/build/npm/registry/
 @fluid-internal:registry=https://pkgs.dev.azure.com/fluidframework/internal/_packaging/build/npm/registry/
 @ff-internal:registry=https://pkgs.dev.azure.com/fluidframework/internal/_packaging/build/npm/registry/
-@ms:registry=https://office.pkgs.visualstudio.com/_packaging/Office/npm/registry/
+@microsoft:registry=https://office.pkgs.visualstudio.com/_packaging/Office/npm/registry/
```

Comment:
I am new npm scopes so this is more a learning question than pointing out something wrong - but what does changing from `ms` to `microsoft` accomplish? It looks [like from this article](https://docs.microsoft.com/en-us/azure/devops/artifacts/npm/scopes?view=azure-devops) the scope simply groups packages together? And the rest of the npmrc doesn't seem to have `ms` as a scope already, so I'm a bit unsure of changing the scope to `microsoft` is accomplishing?

Reply:
The scopes do work as "namespaces" to keep related packages "close by" in the NPM registry, but since they are owned by a user/organization, they can also help with brand identity and trust for consumers. The owner of a scope controls who can publish packages to it (like how we own @fluidframework, @fluid-tools, etc), so if consumers trust the owner user/organization, they know they can probably trust any package in the scope.

About the change from @ms to @microsoft, it's simply that the team at Microsoft that used to publish packages to the @ms scope now publishes them to the @microsoft one, and @ms only contains older versions. And what we're doing in the `.npmrc` file is tell NPM that when it needs to download a package in the @microsoft scope (because it found it as a dependency for something else, or whatever reason), it shouldn't go to the public NPM registry/feed (the one at the top of `.npmrc`, which is the default because there's no scope prefix), but to the one at the URL we specify (and same thing for @fluidframework, @fluid-internal, @ff-internal...) **and** that it must authenticate to it (last line).

Reply:
Thank you for writing this all up! This fully explains the changes!